compiler/parser/parser.cpp

```diff
@@ -414,7 +414,19 @@ static void ensureRequiredStandardModulesAreParsed() {
       UnresolvedSymExpr* oldModNameExpr = toUnresolvedSymExpr(moduleExpr);
 
       if (oldModNameExpr == NULL) {
-        continue;
+        // Allow us to find symbols due to nested references
+        if (CallExpr* call = toCallExpr(moduleExpr)) {
+          if (call->isNamedAstr(astrSdot) &&
+              isUnresolvedSymExpr(call->get(1))) {
+            oldModNameExpr = toUnresolvedSymExpr(call->get(1));
```

Comment:
To avoid the repeated `*UnresolvedSymExpr()` calls (which essentially do the same thing twice, I'd probably refactor this to:

```chapel
if (CallExpr* call = toCallExpr(moduleExpr)) {
  UnresolvedSymExpr* use = toUnresolvedSymExpr(call->get(1));
  if (call->isNamedAstr(astrSdot) && use) {
    oldModNameExpr = use;
  }
}
```

I might also avoid the duplicate else...continue cases by following the conditional up with:

```chapel
 if (oldModNameExpr == NULL) {
  continue;
}
```

(which would also reduce the diff with the previous version).